edgelet/edgelet-core/src/error.rs

```diff
@@ -35,7 +35,9 @@ pub enum ErrorKind {
     Identity,
     #[fail(display = "Error activating key")]
     Activate,
-    #[fail(display = "Edge runtime module has not been created in IoT Hub")]
+    #[fail(
+        display = "Edge runtime module has not been created in IoT Hub (Is this an Edge device?)"
```

Comment:
Suggested text - 
Edge runtime module has not been created in IoT Hub. Please make sure this device is an IoT Edge capable device.

Reply:
Got it, sounds good.